identify-layer-features/src/main/java/com/esri/arcgismaps/sample/identifylayerfeatures/components/MapViewModel.kt

```diff
@@ -71,14 +74,25 @@ class MapViewModel(
         }
 
         // add the world cities layer with and the damaged properties feature layer
-        map.operationalLayers.addAll(listOf(mapImageLayer, featureLayer))
+        map.apply {
+            // set initial Viewpoint to North America
+            initialViewpoint = Viewpoint(
+                center = Point(
+                    x = -10977012.785807,
+                    y = 4514257.550369,
+                    spatialReference = SpatialReference(wkid = 3857)
+                ),
+                scale = 68015210.0
+            )
```

Comment:
This viewpoint feels way tooo precise than it needs to be for this sample. Can we use the simple North America viewpoint as the other [samples](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/blob/4c3001260c2c047ec5b9496bd7a47dc732a6015f/manage-operational-layers/src/main/java/com/esri/arcgismaps/sample/manageoperationallayers/components/MapViewModel.kt#L66):
```suggestion
            // set initial Viewpoint to North America
            initialViewpoint = Viewpoint(39.8, -98.6, 5e7)
```